src/components/display-plants.js

```diff
@@ -1,60 +1,38 @@
 import React from "react"
 import hash from "object-hash"
 
-const DisplayPlants = ({ plants, plantImage }) => {
+const DisplayPlants = ({ plants, plantImage, stateSlug }) => {
   function formatNum(number) {
     return Math.round(number).toLocaleString()
   }
 
   return (
-    <div className="row mt-4 pl-2 pr-2">
+    <div className="display-plants mt-4">
       {plants.map((plant, index) => {
-        // The number of power plants we have to be over to start cutting off -
```

Comment:
We now render _all_ power plants, rather than a limited set. We'll later likely figure out a FE pagination solution but that's a bit too complex to squeeze in here and I didn't notice huge performance issues with the current approach.